CHANGELOG.md

```diff
@@ -1,13 +1,18 @@
-# v4.0.1
+# v5.0.0
 #### Notes
 This release adds the [TESTING.md](TESTING.md) document.
 
+#### Breaking changes
+1. The PYTHONPATH variable no longer imports the oneview module_utils, now a new ENV variable must be set for the modules to work, the ANSIBLE_MODULE_UTILS variable.
```

Comment:
This comment is weird, it seems the ENV var is importing something, not the other way (i.e. something is using the ENV var). Maybe try to write this in the opposite order, like?

_"The new ANSIBLE_MODULE_UTILS environment variable must be set for the modules to work, the PYTHONPATH variable will no longer be used by the oneview module_utils."_

Reply:
If you want, your PR description about this was better than this actual entry on the CHANGELOG:

> This change breaks compatibility with previous versions since having the PYTHONPATH variable setup now won't do anything, and the user will have to setup the ANSIBLE_MODULE_UTILS variable instead.